sos/policies/distros/uniontech-server.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.distros.redhat import RedHatPolicy, OS_RELEASE
+import os
+
+
+class UnionTechPolicy(RedHatPolicy):
+    distro = "Union Tech Linux"
+    vendor = "The UnionTech Project"
+    vendor_urls = [('Distribution Website', 'https://www.chinauos.com/')]
+
+    def __init__(self, sysroot=None, init=None, probe_runtime=True,
+                 remote_exec=None):
+        super(UnionTechPolicy, self).__init__(sysroot=sysroot, init=init,
+                                              probe_runtime=probe_runtime,
+                                              remote_exec=remote_exec)
+
+    @classmethod
+    def check(cls, remote=''):
+
+        if remote:
+            return cls.distro in remote
```

Comment:
Does `/etc/os-release` actually contain the string `Union Tech Linux`, or should this be looking for `UnionTech` like the check below?

Reply:
![image](https://user-images.githubusercontent.com/62841836/127096500-810ed409-b9ba-45cb-ad84-63bccb8473e9.png)

Reply:
Hyphens are problematic (although not outright forbidden) in Python filenames. This is because they are ambiguous in source code with respect to the subtraction operator (python does not strictly require whitespace around this operator, so "import foo-bar" could be interpreted as "import the module named foo-bar" or "import the module name resulting from subtracting bar from foo". For this reason they are highly discouraged - consider using "uniontechserver.py" instead.

Reply:
Right, so for the `remote` check you'd want to check for `UnionTech`, or perhaps `UnionTech OS`, rather than `Union Tech Linux` . The `distro` value can stay the same so banner messages are correct, but this particular check needs to actually match the file content in order for `collect` to be able to work for remote nodes running this distro.

Reply:
@bmr-cymru @TurboTurtle You are right,I accept your suggestion。Thank you very much again.^_^